### PR TITLE
authenticate

### PR DESCRIPTION
**Pull request by namuetn**
## Related Tickets:
- [#28923](https://edu-redmine.sun-asterisk.vn/issues/28923)

## WHAT this PR does?
- ex: Create auth and i8n

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by namuetn** [comment]
ready

File: resources/js/components/ExampleComponent.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-8">
+                <div class="card">
+                    <div class="card-header">Example Component</div>
+
+                    <div class="card-body">
+                        I'm an example component.
+                    </div>
```
Comment (huyenttm-0901) [inline resources/js/components/ExampleComponent.vue]: file này có dùng ko?

File: resources/lang/en/lang.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+return [
+    // login and register
+    'login'=> 'Login',
+    'password'=> 'Password',
+    'register'=> 'Register',
+    'home'=> 'Home',
+    'username_or_mail'=> 'Email address',
+    'save_password'=> 'Save Password',
```
Comment (huyenttm-0901) [inline resources/lang/en/lang.php]: chia file ra em

File: resources/views/auth/passwords/confirm.blade.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                        <div class="form-group row mb-0">
+                            <div class="col-md-8 offset-md-4">
+                                <button type="submit" class="btn btn-primary">
+                                    {{ __('Confirm Password') }}
+                                </button>
+
+                                @if (Route::has('password.request'))
+                                    <a class="btn btn-link" href="{{ route('password.request') }}">
+                                        {{ __('Forgot Your Password?') }}
+                                    </a>
```
Comment (huyenttm-0901) [inline resources/views/auth/passwords/confirm.blade.php]: dùng trans

File: resources/views/auth/passwords/email.blade.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                        <div class="alert alert-success" role="alert">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    <form method="POST" action="{{ route('password.email') }}">
+                        @csrf
+
+                        <div class="form-group row">
+                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('E-Mail Address') }}</label>
```
Comment (huyenttm-0901) [inline resources/views/auth/passwords/email.blade.php]: i18n

File: resources/views/includes/header.blade.php [hunk 5]
```diff
@@ -1,166 +1,167 @@
...
+                                                    <i class="ti-close"></i>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td class="si-pic"><img src="{{ asset('bower_components/bower_fashi_shop/img/select-product-2.jpg') }}" alt=""></td>
+                                                <td class="si-text">
+                                                    <div class="product-selected">
+                                                        <p>$60.00 x 1</p>
+                                                        <h6>Kabino Bedside Table</h6>
+                                                    </div>
```
Comment (huyenttm-0901) [inline resources/views/includes/header.blade.php]: i18n

**Comment by namuetn** [comment]
ready

**Pull request merged**